docs/resources/advanced_cluster.md

```diff
@@ -681,7 +679,7 @@ In addition to all arguments above, the following attributes are exported:
     - DELETING
     - DELETED
     - REPAIRING
-* `replication_specs` - List of settings that configure your cluster regions. This array has one object per shard representing node configurations in each shard. For replica sets there is only one object representing node configurations. If for each replication_spec `num_shards` is configured >1 (i.e. using legacy sharding schema), then each object represents a zone with one or more shards.  Primary usage is covered under the [replication_specs argument reference](#replication_specs), though there are some computed attributes:
+* `replication_specs` - List of settings that configure your cluster regions. This array has one object per shard representing node configurations in each shard. For replica sets there is only one object representing node configurations. If for each replication_spec `num_shards` is configured > 1 (using legacy sharding schema), then each object represents a zone with one or more shards.  Primary usage is covered under the [replication_specs argument reference](#replication_specs), though there are some computed attributes:
```

Comment:
Instead of Primary usage is covered, break this sentence into two:
To learn more, see link.
This setting relies on the following computed attributes: